lib/AttemptResult.d.ts

```diff
@@ -6,23 +6,28 @@ import { RateLimit } from "./RateLimit";
 export interface AttemptResult {
     /**
      * The number of requests this rate limit allows per time window
+     * @readonly
      */
-    limit: number;
+    readonly limit: number;
     /**
      * The number of requests remaining in the current time window
+     * @readonly
      */
-    remaining: number;
+    readonly remaining: number;
     /**
      * The number of seconds until the current time window resets
+     * @readonly
      */
-    reset: number;
+    readonly reset: number;
     /**
      * The rate limit that this attempt was made on
+     * @readonly
      */
-    rateLimit: RateLimit;
+    readonly rateLimit: RateLimit;
     /**
      * Whether this attempt should be allowed to proceed. If false, the attempt is rate limited.
+     * @readonly
      */
-    allow: boolean;
+    readonly allow: boolean;
```

Comment:
Small comment, sometimes your IDE may suggest `readonly` but please be careful since in some cases it may suggest to a value you may change.

Reply:
AttemptResult returns data about the rate limit attempt. It should not be modified